microsoft-edge/webview2/release-notes.md

```diff
@@ -35,6 +35,72 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
 To use a prerelease SDK along with a Microsoft Edge preview channel, go to [Switch to a preview channel to test upcoming APIs and features](how-to/set-preview-channel.md).
 
 
+<!-- ====================================================================== -->
+## 1.0.1108.44
+
+Release Date: February 6, 2022
+
+[NuGet package for WebView2 SDK 1.0.1108.44](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1108.44)
+
+For full API compatibility, this version of the WebView2 SDK requires WebView2 Runtime version 98.1108.44 or higher.
+
+### General
+
+#### Promotions
+
+The following items are now stable:
+
+*  The [Navigation API](/microsoft-edge/webview2/reference/win32/icorewebview2navigationstartingeventargs2?view=webview2-1.0.1108.44&preserve-view=true) that provides ability to provide additional allowed frame ancestors.
```

Comment:
Is this a duplicate of the below 'add_NavigationStarting' api for iFrames? I don't think we added any modfication to our non iframe navigation api? @ViktoriyaBryhider would know.

Reply:
It is AdditionalAllowedFrameAncestors https://docs.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/icorewebview2experimentalnavigationstartingeventargs?view=webview2-1.0.1083-prerelease - does not look like dupe.